05_syntax_lab/03.py

```diff
@@ -1,7 +1,13 @@
-"""
-Write a program that randomizes a number
-and prints the sum total of its digits.
-For example if the number was: 2345
-The result should be: 14
-"""
-
+from random import randint
+"""
+Print the sum of a the digits of a random number between 1-10000
+"""
+
+
+input=str(randint(1,10000))

```

Comment:
Notice how github colours "input" in the same color as print (and differently from "onechar"). The reason is python has a function named input, so when you use that as a variable name it can be a bit confusing.